config.md

```diff
@@ -265,6 +265,27 @@ Annotations are key-value maps.
 }
 ```
 
+## HypervisorPath
```

Comment:
I think we might want to consider adding ``RuntimePath``. Currently, ``dockerd`` will launch ``containerd`` and ``containerd`` will launch the runtime configured when it was started.

But this implies that if a single host is to support multiple runtimes, *you would need two instances of* ``containerd``:

* ``containerd --runtime foo --state-dir /run/containerd-foo --listen unix:///run/containerd/containerd-foo.sock --runtime /path/to/foo/runtime``

* ``containerd --runtime bar --state-dir /run/containerd-bar --listen unix:///run/containerd/containerd-bar.sock --runtime /path/to/bar/runtime``

But, *you would also need two instances of* ``dockerd``, one to talk to each of the above:

* ``dockerd --containerd /path/to/containerd1.sock``
* ``dockerd --containerd /path/to/containerd2.sock``

This doesn't sound ideal.

But, if we were to introduce ``RuntimePath``, only a single ``dockerd`` and a single ``containerd`` instance would be required. We could then remove ``--runtime`` and ``--runtime-args`` from ``containerd`` and have it launch the runtime specified by ``RuntimePath`` (we'd also need a ``RuntimeArgs`` of course).

Maybe another way to handle this is to allow ``RuntimePath`` to be specified. If it is, ``containerd`` (for example) would honour that (if the path existed). If not specified ``containerd `` would default to the value of ``--runtime`` (or "``runc``" if ``--runtime`` is not specified).